src/networks/avalanche.ts

```diff
@@ -0,0 +1,68 @@
+import { getProviderForNetwork, proxyRpcEndpoint } from '@/handlers/web3';
+import { Network, NetworkProperties } from './types';
+import { gasUtils } from '@/utils';
+import { avalanche } from '@wagmi/chains';
+import { AVAX_AVALANCHE_ADDRESS } from '@/references';
+import { getAvalancheGasPrices } from '@/redux/gas';
+import { getRemoteConfig } from '@/model/remoteConfig';
+
+export const getAvalancheNetworkObject = (): NetworkProperties => {
+  const { avax_enabled, avax_tx_enabled } = getRemoteConfig();
+  return {
+    // wagmi chain data
+    ...avalanche,
+
+    // network related data
+    enabled: avax_enabled || __DEV__,
```

Comment:
if you decide to switch `avax` -> `avalanche` it'll need to be updated here

Reply:
can we also remove dev here? doesnt match the others